x-pack/test_utils/README.md

```diff
@@ -0,0 +1,255 @@
+# Testbed utils
+
+The Testbed is a small library to help testing React components. It is most usefull when testing application "sections" (or pages) in **integration** 
+than when unit testing single components in isolation.
+
+## Motivation
+
+We dediced (the Elasticsearch UI team) to build this small abstraction to help with our "client integration" testing. When testing complete "pages" 
```

Comment:
This can just be "The Elasticsearch UI team built this to support client-side integration testing."